compute/gpu/troubleshooting/fix-driver-issues.mdx

```diff
@@ -0,0 +1,23 @@
+---
+meta:
+  title: Fixing GPU issues after installing nvidia-driver packages
+  description: This section provides information how to fix GPU issues after installing nvidia-driver packages
+content:
+  h1: Fixing GPU issues after installing nvidia-driver packages
+  paragraph:  This section provides information how to fix GPU issues after installing nvidia-driver packages
+tags: gpu issue error driver nvidia-driver sleep gpu-instance
+dates:
+  validation: 2022-12-01
+  posted: 2022-12-01
+categories:
+  - compute
+---
+
+All Scaleway [GPU Instances](/compute/gpu/quickstart/) come with a Nvida driver pre-installed. This means you do not have to install them by yourself.
+The drivers installed on each Instance are from the `nividia-headless-<driver version>` packages. These are intended for use on cloud Instances without a Windows Desktop or display server. 
+
+<Message type="important">
+    The manual installation of a `nvidia-driver` package forces the installation of the `xorg` display server and other Gnome packages. As there is no physical display connected to the GPU of your Instance, this may cause side-effects such as your Instance entering into sleep mode after a while, resulting in the user being disconnected.
```

Comment:
```suggestion
    The manual installation of a `nvidia-driver` package forces the installation of the `xorg` display server and other Gnome packages. As there is no physical display connected to the GPU of your Instance, this may cause side-effects such as your Instance entering into sleep mode after a while. The user will get disconnected as a result.
```